cgmml/common/depthmap_toolkit/tests/test_depthmap_toolkit_utils.py

```diff
@@ -1,6 +1,31 @@
 import numpy as np
 
-from cgmml.common.depthmap_toolkit.depthmap_utils import matrix_transform_point
+from cgmml.common.depthmap_toolkit.depthmap_utils import calculate_boundary, get_smoothed_pixel, matrix_transform_point
+
+
+def test_calculate_boundary():
+    input = np.array([
+        [0, 0, 0, 0],
+        [0, 1, 1, 0],
+        [0, 1, 1, 0],
+        [0, 0, 0, 0],
+    ])
+    expected = np.array([1, 1, 2, 2])
+    np.testing.assert_array_equal(expected, calculate_boundary(input))
+
+
+def test_get_smoothed_pixel():
+    black = [0., 0., 0.]
+    white = [1., 1., 1.]
+    rgb = np.array([
+        [black, black, black, black, black],
+        [black, black, black, black, black],
+        [black, black, white, black, black],
+        [black, black, black, black, black],
+        [black, black, black, black, black],
+    ])
```

Comment:
This should be equivalent

```suggestion
    rgb = np.zeros((5, 5, 3))  # black
    rgb[2, 2] = 1.  # white
```